vividus/src/main/java/org/vividus/bdd/expression/HashExpressionProcessor.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.expression;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.inject.Named;
+
+import org.vividus.bdd.converter.FluentTrimmedEnumConverter;
+import org.vividus.util.ResourceUtils;
+
+@Named
+public class HashExpressionProcessor extends AbstractExpressionProcessor<String>
+{
+    private static final Pattern HASH_PATTERN = Pattern.compile("^(?:calculateHash|calculateFileHash)\\((.*), (.*)\\)$",
```

Comment:
```suggestion
    private static final Pattern HASH_PATTERN = Pattern.compile("^(calculate(?:File)?Hash)\\((.*), (.*)\\)$",
```
it will change group order, but below it will be possible to simplify `if`:
```java
"calculateHash".equalsIgnoreCase(expressionMatcher.group(1))
```